tempo_embeddings/text/highlighting.py

```diff
@@ -5,3 +5,8 @@
 class Highlighting:
     start: int
     end: int
+
+    def get_span(self, stringify: str = False):
+        if stringify:
+            return f"{self.start}_{self.end}"
+        return (self.start, self.end)
```

Comment:
I'm not sure why exactly this method is needed. If it is only for a string representation, you can overwrite the `__str__()` method:

```suggestion

    def __str__(self):
        return f"{self.start}_{self.end}"
```